commit 2b03cfeea47834913ff769124f4deba88140f662
Author: Robert Haas <rhaas@postgresql.org>
Date:   Wed Aug 21 09:58:11 2024 -0400

    Fix pgindent damage
    
    Oversight in commit a95ff1fe2eb4926b13e0940ad1f37d048704bdb0

diff --git a/src/backend/utils/adt/varlena.c b/src/backend/utils/adt/varlena.c
index 973034548d2..f2ecef000a0 100644
--- a/src/backend/utils/adt/varlena.c
+++ b/src/backend/utils/adt/varlena.c
@@ -1920,20 +1920,20 @@ varstr_sortsupport(SortSupport ssup, Oid typid, Oid collid)
 
 		/*
 		 * Unfortunately, it seems that abbreviation for non-C collations is
 		 * broken on many common platforms; see pg_strxfrm_enabled().
 		 *
-		 * Even apart from the risk of broken locales, it's possible that there
-		 * are platforms where the use of abbreviated keys should be disabled at
-		 * compile time.  Having only 4 byte datums could make worst-case
-		 * performance drastically more likely, for example.  Moreover, macOS's
-		 * strxfrm() implementation is known to not effectively concentrate a
-		 * significant amount of entropy from the original string in earlier
-		 * transformed blobs.  It's possible that other supported platforms are
-		 * similarly encumbered.  So, if we ever get past disabling this
-		 * categorically, we may still want or need to disable it for particular
-		 * platforms.
+		 * Even apart from the risk of broken locales, it's possible that
+		 * there are platforms where the use of abbreviated keys should be
+		 * disabled at compile time.  Having only 4 byte datums could make
+		 * worst-case performance drastically more likely, for example.
+		 * Moreover, macOS's strxfrm() implementation is known to not
+		 * effectively concentrate a significant amount of entropy from the
+		 * original string in earlier transformed blobs.  It's possible that
+		 * other supported platforms are similarly encumbered.  So, if we ever
+		 * get past disabling this categorically, we may still want or need to
+		 * disable it for particular platforms.
 		 */
 		if (!pg_strxfrm_enabled(locale))
 			abbreviate = false;
 	}
 
